test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -188,6 +486,125 @@ static public void testCreateTriangle2D() throws Throwable {
+		assertEquals(array[1].st.x, line2D_2.st.x);
+		assertEquals(array[0].st.y, line2D_1.st.y);
+		assertEquals(array[1].st.y, line2D_2.st.y);
+		assertEquals(array[0].en.x, line2D_1.en.x);
+		assertEquals(array[1].en.x, line2D_2.en.x);
+		assertEquals(array[0].en.y, line2D_1.en.y);
+		assertEquals(array[1].en.y, line2D_2.en.y);
+	}
+
+	@Test(priority=4, invocationCount=2)
+	static public void testCreateArrayTriangle2D() throws Throwable {
+		Triangle2D[] array = new Triangle2D[10];
+		Triangle2D triangle1 = new Triangle2D(defaultTrianglePositions);
+		Triangle2D triangle2 = new Triangle2D(defaultTrianglePositionsNew);
+		Triangle2D triangleEmpty = new Triangle2D(defaultTrianglePositionsEmpty);
+
+		array[0] = triangle1;
+		array[1] = triangleEmpty;
+		array[2] = triangle2;
+		array[3] = triangleEmpty;
+		array[4] = triangle1;
+		array[5] = triangleEmpty;
+		array[6] = triangle2;
+		array[7]  = triangleEmpty;
+		array[8] = triangle1;
+		array[9] = triangleEmpty;
+
+		checkEqualTriangle2D(array[0], defaultTrianglePositions);
+		checkEqualTriangle2D(array[1], defaultTrianglePositionsEmpty);
+		checkEqualTriangle2D(array[2], defaultTrianglePositionsNew);
+		checkEqualTriangle2D(array[3], defaultTrianglePositionsEmpty);
+		checkEqualTriangle2D(array[4], defaultTrianglePositions);
+		checkEqualTriangle2D(array[5], defaultTrianglePositionsEmpty);
+		checkEqualTriangle2D(array[6], defaultTrianglePositionsNew);
+		checkEqualTriangle2D(array[7], defaultTrianglePositionsEmpty);
+		checkEqualTriangle2D(array[8], defaultTrianglePositions);
+		checkEqualTriangle2D(array[9], defaultTrianglePositionsEmpty);
+	}
+
+	@Test(priority=1)
+	static public void testCreateFlattenedValueInt() throws Throwable {
+		int i = Integer.MAX_VALUE;
+		ValueInt! valueInt = new ValueInt(i);
+		assertEquals(valueInt.i, i);
+		// TODO add putfield tests once withfield is replaced
+	}
+
+	@Test(priority=1)
+	static public void testCreateFlattenedValueLong() throws Throwable {
+		long l = Long.MAX_VALUE;
+		ValueLong! valueLong = new ValueLong(l);
+		assertEquals(valueLong.l, l);
+		// TODO add putfield tests once withfield is replaced
+	}
+
+	@Test(priority=1)
+	static public void testCreateFlattenedValueDouble() throws Throwable {
+		double d = Double.MAX_VALUE;
+		ValueDouble! valueDouble = new ValueDouble(d);
+		assertEquals(valueDouble.d, d);
+		// TODO add putfield tests once withfield is replaced
+	}
+
+	@Test(priority=1)
+	static public void testCreateFlattenedValueFloat() throws Throwable {
+		float f = Float.MAX_VALUE;
+		ValueFloat! valueFloat = new ValueFloat(f);
+		assertEquals(valueFloat.f, f);
+		// TODO add putfield tests once withfield is replaced
+	}
+
+	@Test(priority=1)
+	static public void testCreateFlattenedValueObject() throws Throwable {
+		Object val = (Object)0xEEFFEEFF;
+		ValueObject! valueObject = new ValueObject(val);
+		assertEquals(valueObject.val, val);
+		// TODO add putfield tests once withfield is replaced
+	}
+
```

Comment:
`position` (instead of `positions`) looks like a more reasonable name for the second input parameter of `checkEqualPoint2D()` below.